workflow_scripts/update_onnx_hub_manifest.py

```diff
@@ -0,0 +1,122 @@
+import glob
+import hashlib
+import json
+import os
+import re
+import bs4
+import markdown
+import pandas as pd
+import typepy
+from os.path import join, split
+
+
+# Acknowledgments to pytablereader codebase for this function
+def parse_html(table):
+    headers = []
+    data_matrix = []
+    rows = table.find_all("tr")
+    re_table_val = re.compile("td|th")
+    for row in rows:
+        td_list = row.find_all("td")
+        if typepy.is_empty_sequence(td_list):
+            if typepy.is_not_empty_sequence(headers):
+                continue
+            th_list = row.find_all("th")
+            if typepy.is_empty_sequence(th_list):
+                continue
+            headers = [row.text.strip() for row in th_list]
+            continue
+        data_matrix.append(list(row.find_all(re_table_val)))
+
+    if typepy.is_empty_sequence(data_matrix):
+        raise ValueError("data matrix is empty")
+
+    return pd.DataFrame(data_matrix, columns=headers)
+
+
+markdown_files = list(glob.glob(join("..", "**", "README.md"), recursive=True))[1:]

```

Comment:
Explicitly scan for the "vision" and "text" folder:

```suggestion
globDirs = [join("..", "vision"), join("..", "text")]
markdown_files = []
for dir in globDirs:
    markdown_files += glob.glob(join(dir, "**", "README.md"), recursive=True)
```